fastccd_support_ioc/delay_generator_ioc.py

```diff
@@ -25,72 +33,98 @@ class DelayGenerator(PVGroup):
     """
 
     trigger_rate = pvproperty_with_rbv(dtype=float, doc="TriggerRate")
-    trigger_on_off = pvproperty_with_rbv(dtype=float, doc="TriggerOnOFF")
+    trigger_on_off = pvproperty_with_rbv(dtype=bool, doc="TriggerOnOFF")
```

Comment:
bool does not work for me to actually change the trigger mode, since I did not know to fix it otherwise, I used int. Which is not great because we only need 0 an 1.

Reply:
Seems like its working for me. Can you confirm or share the issue?